src/main/java/com/rmn/qa/aws/AwsVmManager.java

```diff
@@ -493,8 +493,11 @@ String getNodeConfig(final String uuid, final String hostName, final String brow
         Date createdDate = Calendar.getInstance().getTime();
 
         // Pass in the created date so we can know when this node was spun up
-        nodeConfig = nodeConfig.replaceAll("<CREATED_DATE>", AwsVmManager.NODE_DATE_FORMAT.format(createdDate));
-        nodeConfig = nodeConfig.replaceFirst("<HOST_NAME>", hostName);
+        String nodeCreatedDate = AwsVmManager.NODE_DATE_FORMAT.format(createdDate);
```

Comment:
So I have this code working locally with Selenium 3.0, and I'm not sure why you're having to put quotes in here?  Here is what I have locally that works:
```
        nodeConfig = nodeConfig.replaceAll("<CREATED_DATE>", AwsVmManager.NODE_DATE_FORMAT.format(createdDate));
        nodeConfig = nodeConfig.replaceFirst("<HOST_NAME>", hostName);
```